logins-api/android/LoginsApi/loginsapi/src/main/java/org/mozilla/loginsapi/SyncResult.kt

```diff
@@ -0,0 +1,241 @@
+package org.mozilla.loginsapi
+
+import java.util.ArrayList
+
+// TODO: Unfortunately, this is copied from fxa-result in mozilla-mobile (which AIUI is based on
+// GeckoResult?). It would be nice to have a better way to avoid this issue that
+// doesn't add a dependency on the fxa lib to the sync lib (or is that ok?)
+
+/**
+ * SyncResult is a class that represents an asynchronous result.
+ *
+ * @param <T> The type of the value delivered via the SyncResult.
+ */
+class SyncResult<T> {
```

Comment:
I thought we decided it probably wasn't necessary to clone all of this for sync? Or maybe you caught up with Ed out of band about this?

Reply:
No, I'm not sure I was part of that discussion. It's not clear what subset we'd clone though, since it all seems like things we want callers to be able to do... Unless we should just take a dependency on the FxA lib.

Reply:
"Firefox Sync 1.5". I'm open to other names though!